docs/teams/list-teams.md

```diff
@@ -0,0 +1,19 @@
+---
+title: List Teams
+intro: View your list of teams
+links:
+  overview:
+  quickstart:
+  previous: teams/add-team
+  next: teams/view-team
+  guides:
+  related:
+    - teams/add-team
+  featured:
+---
+
+1. When accessing Devopness, existing projects are automatically listed on the home screen. To navigate between projects, click the Devopness logo in the top-left corner.
+2. Select a project.
+3. Find the Teams card.
+4. If you don't have any teams yet, create one by clicking the Add team button. You can also learn how to create a team on the Add a team documentation page.
+5. Click view all on the Teams card to see a list of existing teams.
```

Comment:
Shouldn't this be step 4
```
3. Find ... card ...
4. Click view all ...
```
?

The note about "If you don't have ... click to add ..." will only make sense for those that already clicked to see the full list, right?

More over, if the UI is well designed, the `ADD TEAM` button should be visible and clear enough for users to understand that there is an option to add the first one, so perhaps that's beyond the scope of this topic that focus on tell user how to list teams?

A more pragmatic and straight to the point example can be seen at https://www.devopness.com/docs/environments/list-environments